src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsFound() {
+        
+        assertTrue("Single null object should fail", Utils.isAnyNull((Object) null));
+        
+        assertTrue("Multiple null objects should fail", Utils.isAnyNull((Object) null, (Object) null, (Object) null));
+    
+        assertTrue("Single null object at the end of non-null objects", Utils.isAnyNull((Object) new String(), (Object) new String(), (Object) null));
+    
+        assertTrue("Single null object in the middle of non-null objects", Utils.isAnyNull((Object) new String(), (Object) null, (Object) new String()));
+        
+        assertFalse("One non-null object", Utils.isAnyNull((Object) new ArrayList<String>()));
+        
+        assertFalse("Many non-null objects", Utils.isAnyNull((Object) new String(), (Object) new Integer(100), (Object) new Double(100.5)));
+    
+        ArrayList<String> list;
+        
+        list = new ArrayList<String>();
+        list.add("Hello");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.